lib/merit/excess.rb

```diff
@@ -2,6 +2,14 @@ module Merit
   class Excess
     include NetLoadHelper
 
+    # Public: total_number_of_events
```

Comment:
Comment formatting is a bit strange: there's no need to repeat the method name (it's an extra thing to change if the method is later renamed). Merit uses [Tomdoc](http://tomdoc.org/) for method comments, so you could simplify this as:

```ruby
# Public: Returns the total number of excess events.
def total_number_of_events
end
```

... or if the return type isn't obvious:

```ruby
# Public: Determines the total number of excess events.
#
# Returns an integer.
def total_number_of_events
end
```